test/fixtures/krb5kdc-fixture/src/main/resources/provision/krb5.conf.template

```diff
@@ -25,7 +26,7 @@
 [realms]
     ${REALM_NAME} = {
         kdc = 127.0.0.1:88
-        kdc = 127.0.0.1:${MAPPED_PORT}
+        #KDC_DOCKER_HOST
```

Comment:
This was required as newer krb5 versions verify the whole conf file before starting and that invalid kdc declaration broke compatibility. This tweak fixes that